### PR TITLE
Add python wrapper for readCalibration.

### PR DESCRIPTION
**Pull request by herr-biber**
Reads .ini or .yaml calibration file and returns camera name and sensor_msgs/cameraInfo.
Includes test.

```
catkin_make run_tests_camera_calibration_parsers
```

### CODE REVIEW

File: camera_calibration_parsers/CMakeLists.txt [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+find_package(PythonLibs 2.7 REQUIRED)
+include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+catkin_python_setup()
 
 catkin_package(
   INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME}
+  LIBRARIES ${PROJECT_NAME} ${PROJECT_NAME}_wrapper
```
Comment (vrabaud) [inline camera_calibration_parsers/CMakeLists.txt]: The wrapper library should not be here as it is not a proper library. Just leave that line unchanged.

**Comment by vrabaud** [comment]
Hi, I added Travis to the repo so you can now check if you PR will pass.
Thx for the contribution though, it's really needed, and you did it the right way ! Thx.

**Comment by herr-biber** [comment]
Hi Vincent,

thanks for the quick reply. I removed the wrapper library from catkin_package(). Travis is happy as well. Do you want me to rebase -i into one single commit?

Cheers,
~markus

**Comment by vrabaud** [comment]
yes, please squash the commits: the rule is usually simple: several commits are allowed in a PR as long as each of them is valid / compiles /passes Travis and helps understanding the breakdown of the PR.
Very nice PR, thx ! Did you try it out with Python 3 by any chance ? (I see you impose 2.7).

**Comment by herr-biber** [comment]
Hi Vincent,
thanks. I squashed the commits. I haven't tried out Python3 yet.

**Pull request merged**